src/components/AutoCompleteTextarea/index.ts

```diff
@@ -0,0 +1,7 @@
+export { default as DefaultSuggestionListItem } from './Item';
```

Comment:
we should remove all the default exports and do `export *` like in other places in the library